### ISSUE TITLE
GM hidden rolls

### ISSUE THREAD

**Issue by Tiduskhyn** [Bubz43/ep2e#5]
Hello,

I wanted to know if there is an option to hide the rolls from GM while still allowing the PCs to be able to see the chat and click on the fray roll or take damage ?
Because if GM is in hidden rolls, nothing appears for them in the chat.

Thank you :)

**Comment by Bubz43** [comment]
There isn't, mostly because I'm not sure what the use case for the option would be. Taking damage or opposing the role both naturally leak the result. So what is the point of hiding the roll part? The way I thought of it was the GM might blind roll for stuff the players just don't know about at all, or to see if they know about something. In the latter case, you can then just publicly reveal the message and they can oppose/take damage as normal.

**Comment by Tiduskhyn** [comment]
The goal is to hide things from the PCs. They don't have to know if an attack roll hit them or not for exemple. But there they see the result. For opposed rolls, GM would prefer roll secretly for the NPCs.
As the character sheet is very cool, the players click directly in the chat box for fray and damages. In this case they necessarily see GM's roll.
I don't change the results often, and usually it's to save the ass of my players.

**Comment by Bubz43** [comment]
I guess just hiding the success test part of the message would be reasonable. It would need to be a new option, as I want blind/whispers to work the same way they do now. Maybe something like `Public [Hide Test]`. This would cause the message to show up the same as if it was rolled publicly, but with the `Roll/Target/Result` section only visible to GMs. 
GM sees
![image](https://user-images.githubusercontent.com/20649078/204105774-ea7e964f-b270-4d04-ac25-ba25611eb230.png)

Players see
![image](https://user-images.githubusercontent.com/20649078/204105808-da4dffcf-a4bf-47b7-98e6-34b126309e47.png)

Is this what you are picturing?

**Comment by Tiduskhyn** [comment]
Yes that's exactly that I would like. The option already exists ?

**Comment by Bubz43** [comment]
No, but should be fairly trivial to add.

**Comment by Tiduskhyn** [comment]
Ok. I'am using an older version of the sheet. So I would do without.
Thanks for the super quick responses.
All my players and I love this super automated sheet.
Nice job.
